test/GitHub/Listener/GitHubReleaseMastodonListenerTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppTest\GitHub\Listener;
+
+use App\GitHub\Event\GitHubRelease;
+use App\GitHub\Listener\GitHubReleaseMastodonListener;
+use App\GitHub\Listener\GitHubReleaseTweetListener;
+use App\Mastodon\MastodonClient;
+use Laminas\Twitter\Response;
+use Laminas\Twitter\Twitter;
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Prophecy\PhpUnit\ProphecyTrait;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Log\LoggerInterface;
+use RuntimeException;
+
+class GitHubReleaseTootListenerTest extends TestCase
+{
+    use ProphecyTrait;
```

Comment:
Don't use prophecy. We need to drop it from other tests too.

Reply:
That was copy/paste! But I'm not very fond of it anyhow, so I'll rewrite the tests.

I would use phpunits mocking framework. Or is there anything else that we are using elsewhere?

Reply:
Just the phpunit mocking.

I am personally fond of prophecy since it allows to easily mock different behaviors or expectations with different argument sets but it has it's downsides too and it lags behind keeping up with the phpunit breaking changes.